SwrveSDKCommon/Common/SwrveCommon.h

```diff
@@ -1,21 +1,20 @@
 #import <UIKit/UIKit.h>
 
 /*! Swrve SDK shared protocol (interface) definition */
-@protocol ISwrveCommon <NSObject>
+@protocol SwrveCommonDelegate <NSObject>
+
+@required
 
 -(NSData*) getCampaignData:(int)category;
--(void) sendQueuedEvents;
--(int) eventInternal:(NSString*)eventName payload:(NSDictionary*)eventPayload triggerCallback:(bool)triggerCallback;
 -(int) userUpdate:(NSDictionary*)attributes;
--(void) setLocationVersion:(NSString*)locationVersion;
 -(BOOL) processPermissionRequest:(NSString*)action;
 
 @end
 
 @interface SwrveCommon : NSObject
 
-+(id<ISwrveCommon>) getSwrveCommon;
-+(void) setSwrveCommon:(id<ISwrveCommon>)swrveCommon;
++(id<SwrveCommonDelegate>) getSwrveCommon;
++(void) setSwrveCommon:(id<SwrveCommonDelegate>)swrveCommon;
```

Comment:
This is essentially a sharedInstance so shouldn't we do something similar to what we do in the main Swrve class:
https://github.com/Swrve/swrve-ios-sdk/blob/release-4_2/Sdk/Track/Swrve.m#L568
[SwrveCommon sharedInstance]
[SwrveCommon addSharedInstance:......]